style.css

```diff
@@ -0,0 +1,27 @@
+ .display{ 
+        width: 22.5em;
+        line-height: 2em;
+         height: 3em; 
+         margin-left: 15px;
+                  }
+                 
+ .clear{
+         width: 4em;
+         padding: 1em;
+        font-weight: bold;
+         background: red;
+       color: white;
+                  }
+
+    .firs{

```

Comment:
use a spell checker in your code editor if you can. It would help you easily spot typo errors